test/extensions/voting-rep.js

```diff
@@ -1525,6 +1530,85 @@ contract("Voting Reputation", (accounts) => {
       const pastVote = await voting.getExpenditurePastVote(slotHash);
       expect(pastVote).to.eq.BN(REQUIRED_STAKE);
     });
+
+    it("transactions that try to execute an allowed method on Reputation Voting extension are accepted by the MTX broadcaster", async function () {
```

Comment:
I'm thinking whether it would be better to keep all the MTX tests inside a single test file which loads in all the extensions etc vs having to put a few similar tests in every test file.

Reply:
These tests require a reputation state, so I opted to have these tests here, rather than copy the `before` that's in this file in to the metatransaction file (which is much larger than the equivalent `before`s I copied in to here).